src/config/networks.ts

```diff
@@ -123,7 +123,7 @@ const makeNetworkList = () => {
     },
     namespace: 'creditcoin-mainnet',
     subscanEndpoint: 'http://127.0.0.1:4399',
-    subscanUrl: 'https://creditcoin3.subscan.io/',
+    subscanUrl: 'https://creditcoin3.subscan.io',
```

Comment:
This need to matches whatever it's on the other networks, I think this one is fine. But just to remember double check it.